PatientMonitoringSystem/PatientMonitoringSystem/BedsideSystemController.cs

```diff
@@ -93,5 +103,16 @@ private void RemoveModule(object sender, OnRemoveModuleEventArgs e)
 
 			addButton.Enabled = true;
 		}
+
+		private IModuleStrategy CreateModuleStrategy(string module_name) {
+			switch (module_name) {
+				case "Blood Pressure":
```

Comment:
We need to have some kind of way to better manage the string usage here.  Perhaps we need to think about using Enums?